src/sentry/notifications/notification_action/utils.py

```diff
@@ -12,11 +13,27 @@
 
 
 def execute_via_group_type_registry(
-    job: WorkflowEventData, action: Action, detector: Detector
+    event_data: WorkflowEventData, action: Action, detector: Detector
 ) -> None:
+    """
+    Generic "notification action handler" this method will lookup which registry
+    to send the notification to, based on the type of detector that created it.
+
+    This currently only supported detector types: 'error', 'metric_issue'
+
+    If an `Activity` model for a `Group` is provided in the event data
+    it will send an activity notification instead.
+    """
+    if isinstance(event_data.event, Activity):
+        # TODO - this is a workaround to ensure a notification is sent about the issue.
+        # We'll need to update this in the future to read the notification configuration
+        # from the Action, then get the template for the activity, and send it to that
+        # integration.
+        return event_data.event.send_notification()
```

Comment:
Do we have test coverage for this?

Reply:
👍 i can add a test real quick.

some additional context is that we'll need to refactor this a bit more. we need the noa to figure out if we should be making a reply to a thread or send this generic notification update.